Taskfile.yml

```diff
@@ -0,0 +1,151 @@
+version : '3'
+
+set: [pipefail]
+
+vars:
+  SPECFILE:
+    sh: if [ -f "{{ .USER_WORKING_DIR }}/package.yml" ]; then echo "package.yml"; else echo "pspec.xml"; fi;
+
+tasks:
+  # Utility tasks, mainly for precondition checks
+  package-file:
+    internal: true
+    desc: |
+      Check that either `package.yml` or `pspec.xml` exists in the current directory
+    dir: '{{ .USER_WORKING_DIR }}'
+    requires:
+      vars: [SPECFILE]
+    preconditions:
+      - sh: test -f package.yml || test -f pspec.xml
+        msg: Either `package.yml` or `pspec.xml` must exists in the current directory
+
+  # Build packages
+  build: 
+    desc: Build the current package against the unstable repo
+    aliases: [default]
+    dir: '{{ .USER_WORKING_DIR }}'
+    vars:
+      PROFILE: '{{ default "unstable-x86_64" .PROFILE }}'
+    deps:
+      - package-file
+    cmds:
+      - sudo solbuild build {{ .SPECFILE }} -p {{ .PROFILE }} {{ .CLI_ARGS }}
+
+  build-stable:
+    desc: Build the current package against the stable repo (do NOT use for official submissions!)
```

Comment:
Do we want to use a [warning prompt](https://taskfile.dev/usage/#warning-prompts) instead?